assets/js/base/utils/test/errors.js

```diff
@@ -17,9 +20,9 @@ describe( 'formatError', () => {
 	} );
 
 	test( 'should format API errors', async () => {
-		const error = await formatError( {
-			json: () => Promise.resolve( { message: 'Lorem Ipsum' } ),
-		} );
+		const mockResponse = new Response( mockResponseBody, { status: 400 } );
```

Comment:
Use Response to pass more realistic argument to formatError. a partial object that has property of `json` is not type safe or close to real usage.